src/clients/client.ts

```diff
@@ -76,21 +80,24 @@ class Client {
    * @memberof MeiliSearch
    * @method getRawIndex
    * @param {string} indexUid The index UID
-   * @returns {Promise<IndexResponse>} Promise returning index information
+   * @returns {Promise<IndexObject>} Promise returning index information
    */
-  async getRawIndex(indexUid: string): Promise<IndexResponse> {
+  async getRawIndex(indexUid: string): Promise<IndexObject> {
     return new Index(this.config, indexUid).getRawInfo()
   }
 
   /**
    * Get all the indexes as Index instances.
    * @memberof MeiliSearch
    * @method getIndexes
+   * @param {IndexesQuery} parameters - Parameters to browse the indexes
```

Comment:
You can also specify the default value like this if you want:

`@param {IndexesQuery} [parameters={}] - Parameters to browse the indexes`

See https://jsdoc.app/tags-param.html#optional-parameters-and-default-values